contracts/netmap/migration_test.go

```diff
@@ -184,7 +181,11 @@ func testMigrationFromDump(t *testing.T, d *dump.Reader) {
 
 	c.CheckUpdateSuccess(t, updPrm...)
 
-	checkNewEpochSubscribers(t, c, balanceHash, containerHash)
+	if prevVersion < 19_000 {
```

Comment:
A part of the problem is that these new CVSs are _too new_. Migrations we have currently in the code are somewhat trivial (maybe except for the balance contract), nothing compared to the notary migration.

I though about picking some older (post-notary, but not the latest) state, but never managed to do that.

Reply:
> I though about picking some older (post-notary, but not the latest) state

i thought about this too. Can we conveniently determine the moments at which past updates were made? It seems that such a chronology can be useful, similar to epochs. But apparently possible to restore it only manually now

Reply:
suggest resolving in a separate issue

Reply:
We can just forget about it for a moment, all of the current (pre-0.22.0) migrations are basically done and new one is tested here, so not a problem.